### PR TITLE
Fix Error in Program timeline Section

### LINKED ISSUE #1411: Issue in program timeline section in ScholarX page
**Describe the bug**
Issue in the project timeline section.
**Issue is visible when you view the project timeline section using smartphones.**

**To Reproduce**
1. Go to 'ScholarX-2022'
2. Scroll down to 'Project Timeline'  
3. See error

**Expected behavior**
Try to fix the issue using HTML tables.

**Screenshots**
Issue
image

image

**Additional context**
After fixing the bug using HTML tables.
fix

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by janithlahirukariyawasam**
## Purpose

The purpose of this PR is to fix #1411 

## Goals

Able to view program timeline section using any device.

## Approach

Used HTML tables.

### Screenshots

fix
image

  
### Preview Link

https://pr-1412-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/2022/index.html [hunk 1]
```diff
@@ -72,47 +72,87 @@
...
+                                    <td> <span>Mentors announced</span></td>
+                                    <td style="text-align: end;"><span class="float-right">15 March, 2022</span></td>
+                                </tr>
+                            </table>
+                        </li>
+                        <li>
+                            <table style="width: 100%;">
+                                <tr>
+                                    <td><span>Student application period open</span></td>
+                                    <td style="text-align: end;"> <span class="float-right">15 March, 2022</span></td>
```
Comment (anjula-sack) [inline scholarx/2022/index.html]: Shall we remove inline CSS? Let's use a class instead.

File: scholarx/2022/index.html [hunk 2]
```diff
@@ -72,47 +72,87 @@
...
+                                </tr>
+                            </table>
+                        </li>
+                        <li>
+                            <table style="width: 100%;">
```
Comment (anjula-sack) [inline scholarx/2022/index.html]: Same here

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/2022/index.html (hunk 2 above)
Comment (janithlahirukariyawasam) [reply scholarx/2022/index.html]: @anjula-sack Removed inline css.

**Review by anjula-sack — APPROVED** [review]
Good job! @janithlahirukariyawasam

**Pull request merged**